op-challenger/fault/factory.go

```diff
@@ -19,14 +19,16 @@ var (
 type MinimalDisputeGameFactoryCaller interface {
 	GameCount(opts *bind.CallOpts) (*big.Int, error)
 	GameAtIndex(opts *bind.CallOpts, _index *big.Int) (struct {
+		GameType  uint8
```

Comment:
Where is the mapping from `GameType` to string? It'd also be really nice to already have the golang version of it.

Reply:
Don't believe y'all have one yet - glad to add it!

Reply:
Done, lmk if that's what you were looking for.

Reply:
that was it